src/go/rpk/pkg/cli/cmd/debug/bundle_test.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+//go:build linux
+// +build linux
+
+package debug
+
+import (
+	"io/ioutil"
+	"math"
+	"math/rand"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestLimitedWriter(t *testing.T) {
+	const block = 4096
+	tests := []struct {
+		name          string
+		limit         int
+		blocksToWrite float64
```

Comment:
I think we could avoid floats and any rounding error they have if the two tests below are:

1) limit 3, blocksToWrite 2
2) limit 1, blocksToWrite 2

which would also remove the need for math.Min

Reply:
I wanted to simulate non-aligned contents, but I think this is fine.